src/main/java/com/squareup/kotlinpoet/ClassName.kt

```diff
@@ -137,12 +138,10 @@ class ClassName private constructor(
       return ClassName(names)
     }
 
-    @JvmStatic fun get(clazz: KClass<*>): ClassName {
-      val qualifiedName = clazz.qualifiedName
-      if (qualifiedName == null) {
-        throw IllegalArgumentException("$clazz cannot be represented as a TypeName")
-      }
-      return ClassName.bestGuess(qualifiedName)
+    @JvmStatic @JvmName("get")
```

Comment:
True. It was just a mechanical refactor to put them on everything. In theory something could give you a KClass in Java just like you can have a Class in Kotlin